mobile/src/components/YoutubeVideo.tsx

```diff
@@ -1,36 +1,47 @@
-import React from 'react';
-import { Platform } from 'react-native';
-import YoutubePlayer from 'react-native-youtube-iframe';
+import React from "react";
```

Comment:
Can you revert this file please? Doesn't look like anything was actually changed but I think your IDE reformatted it anyways.